test/test_titles.py

```diff
@@ -356,6 +357,38 @@ def testPartialCoverage(self):
         titleAlignments.addAlignment(titleAlignment)
         self.assertEqual(0.3, titleAlignments.coverage())
 
+    def testFullBaseCoverage(self):
+        """
+        The baseCoverage method must return the correct result when the title
+        is fully covered by its reads.
+        """
+        hsp1 = HSP(7, subjectStart=0, subjectEnd=5)
+        hsp2 = HSP(8, subjectStart=5, subjectEnd=10)
+        titleAlignments = TitleAlignments('subject title', 10)
+        read = Read('id1', 'AAA')
+        titleAlignment = TitleAlignment(read, [hsp1, hsp2])
+        titleAlignments.addAlignment(titleAlignment)
+        c = Counter([0, 1, 2, 3, 4, 5, 6, 7, 8, 9])
+        self.assertEqual(c, titleAlignments.baseCoverage())
+
+    def testBaseCoverage(self):
+        """
+        The baseCoverage method must return the correct results when the title
```

Comment:
It would be better if two of the reads overlapped. The tests in this class don't show that that works.